src/components/WYSIWYGInput/index.css

```diff
@@ -95,6 +95,15 @@
     .public-DraftEditor-content {
       /* The initial minimum height for the editor content */
       min-height: 100px;
+
+      div {
+        /* If not the first nor last editor element */
+        >*:not(:first-child):not(:last-child) {
+          /* Adds a margin between editor elements */
+          /* The prose class is not capable of applying margins to paragraph sections for intance */
+          @apply my-5;
```

Comment:
Would you perhaps be able to do this styling here https://github.com/perimetre/ui/blob/master/src/presets/default-preset.js#L113?

Honest question, just asking if maybe it's worth doing it there, don't know if you knew about customizing the prose from there

Reply:
Yes I tried adding it there before (following this [https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/styles.js](https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/styles.js)), but it's tricky because paragraphs for instance are wrapped in a couple of divs by the editor, and just adding the margin to them in the presets won't work. The divs somehow eat that margin haha. and if I apply the margin to divs then the lists items also get it because they are also each item wrapped in a div.

So no, unfortunately I tried doing in the presets but it didn't work.

Reply:
Makes sense!  Alright then, thanks for the explanation 💪

Reply:
Thank you my friend for reviewing this for me! 💪